com.dida.TekstReplace.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.dida.TekstReplace",
```

Comment:
`com.dida` doesn't seem to exist

Reply:
just to clarify: the domain on which the app-id is based shall exist. But you didn't answer that question either.

It's a requirement.